Fixtures/Templates/AddTemplate/GoodTemplate/Tests/___NAME___Tests/___NAME___Tests.swift

```diff
@@ -0,0 +1,11 @@
+import XCTest
+@testable import ___NAME___
+
+final class ___NAME___Tests: XCTestCase {
```

Comment:
Same here about wanting to use the module form of the identifier, along with other similar places.

Reply:
Edit: Saw the ___NAME_AS_C99___ further in the code, so that should be used here.